src/docs/Foundations/MagicUnit/code.mdx

```diff
@@ -137,3 +137,13 @@ It basically just do the `to-rem()` part for you
 ### The shorthands variables :
 
 > the **shorthands variables** behave exactly as the magic-unit-rem, but in a variable form factor.
+
+### The magic-unit grid preview :
```

Comment:
the magic unit grid development utility (to make clear from the start that it is a dev tool)

Reply:
fixed